content/client-lib-development-guide/versions/features-0-8__0-9.diff

```diff
@@ -31,16 +31,16 @@ index bff9943..5f87d5d 100644
 -As a result, this document outlines the complete feature set of both the REST and Realtime client libraries.  It is expected that every client library developer refers to this document to ensure that their client library provides the same API and features as the existing Ably client libraries. In addition to this, it is essential that there is test coverage over all of the features described below.
```

Comment:
What is the difference here, this is a confusing change... Whitespace maybe? Character encoding?

Reply:
I'll have a look later. I'm not sure what the .diff files are or how they are generated.

Reply:
I think these `.diff` files are created by `nanoc` when it does the doc versioning. I don't think we should be modifying these files should we, as I think they are machine generated? To be honest I'm not sure about these files. Would @fliptopbox know?

Reply:
@tomczoink could you help us here with a quick look at what is happening on this line, and also where the `.diff` files come from?

Reply:
This version was a bit before me unfortunately, but I think it's partially hand-edited but not 100% sure.

Reply:
Yup, don't change diff files, they are auto-generated and this will create confusion as this is a record of what changed at that time, so we can't change it after that.

Reply:
I removed the diffs from this change